src/app/series/directives/file-template.component.ts

```diff
@@ -1,33 +1,54 @@
 import { Component, Input } from '@angular/core';
-import { AudioFileTemplateModel } from '../../shared';
+import { AudioVersionTemplateModel, AudioFileTemplateModel } from '../../shared';
 
 @Component({
   selector: 'publish-file-template',
   styleUrls: ['file-template.component.css'],
   template: `
-    <div *ngIf="file && !file.isDestroy">
+    <div *ngIf="file && !file.isDestroy" class="file">
 
-      <publish-fancy-field textinput small required [model]="file" name="label" label="Segment {{file.position}} Label">
-        <span class="fancy-hint">A name for this audio segment, such as "Billboard" or "Part A"</span>
-      </publish-fancy-field>
-
-      <publish-fancy-field small class="length" [model]="file" label="Segment length in seconds" invalid="lengthAny">
-      <span class="fancy-hint">The minimum and maximum durations in seconds for this segment</span>
-
-        <publish-fancy-field number small inline hideinvalid [model]="file" name="lengthMinimum" label="Minimum">
+      <div class="label">
+        <publish-fancy-field textinput required hideinvalid [model]="file" name="label">
         </publish-fancy-field>
+      </div>
 
-        <publish-fancy-field number small inline hideinvalid [model]="file" name="lengthMaximum" label="Maximum">
-        </publish-fancy-field>
+      <div class="lengths">
+        <publish-fancy-duration [model]="file" tiny="true" name="lengthMinimum" label="Min"></publish-fancy-duration>
+        <publish-fancy-duration [model]="file" tiny="true" name="lengthMaximum" label="Max"></publish-fancy-duration>
+      </div>
 
```

Comment:
I love the one-liner matching the story audio uploader -- I think it makes it make much more intuitive sense to user on both pages. But the styling starts to look weird as it wraps onto next line as screen shrinks: 

<img width="485" alt="screen shot 2017-01-24 at 11 25 42 am" src="https://cloud.githubusercontent.com/assets/16007382/22256339/69ab82e8-e228-11e6-8f4c-0f3dda902074.png">

maybe this just shouldn't wrap?